config.json

```diff
@@ -309,6 +309,8 @@
     "color_badge_product_sold_out_badges": "#007dc6",
     "color_text_product_sold_out_badges": "#ffffff",
     "color_hover_product_sold_out_badges": "#000000",
+    "tooltip-textColor--focus": "#ffffff",
+    "tooltip-backgroundColor--focus": "#313440",
```

Comment:
config.json file includes page builder setting, so we need to define the approach we use here:
- using internal scss var 
- implement full cycle of customising with page builder